src/components/game/GameDetails.js

```diff
@@ -0,0 +1,89 @@
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+import Button from 'material-ui/Button'
+import Card, { CardActions, CardContent } from 'material-ui/Card'
+import Typography from 'material-ui/Typography'
+import {withRouter} from 'react-router'
+import {joinGame} from '../../actions/games'
+import './GameDetails.css'
+
+class GameDetails extends PureComponent {
+
+  handleClick = () => {
+    const {game,history,players} = this.props
+    switch(this.buttonText(players)) {
+      case "join":
+        this.props.joinGame(game.id)
+        history.push(`/games/${game.id}`)
+        break
+      case "enter":
+      case "watch":
+        history.push(`/games/${game.id}`)
+        break
+      default:
+        return null
+    }
+  }
+  buttonText = (players) => {
+    const id = this.props.userId
+    const game = this.props.game
+    if (players.includes(id)) {
+      if (game.status === "started") return "enter"
+      return "..."
```

Comment:
I have an idea for this one, allow the player to leave the game if it's still pending. Then we delete that game! But we can leave that for later, add it to the todo list haha

Reply:
so like leaving it without returning back to it?
yeah to do list haha 
did we actually write down those things somewhere? 😅

Reply:
Yes thing like that! 
Haha I though you have them somewhere :stuck_out_tongue: 
Well we can go through all reviews to find them :joy:

Reply:
I can try to make a list haha oops damn it